.eslintrc.yml

```diff
@@ -14,3 +14,4 @@ rules:
   unicorn/prefer-array-find: 0
   unicorn/no-abusive-eslint-disable: 0
   unicorn/no-lonely-if: 0
+  unicorn/no-array-for-each: 0
```

Comment:
Can you disable this in the specific file instead of for the whole repo?

Reply:
Thank you for catching all my dumb mistakes and the other suggestions are all good but forEach is good and these unicorn rules are overly restrictive

Reply:
in order to be constructive and not just arguing, look at the [justifications](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md). Performance - doesnt apply, readability - i would argue for of is less readable but i think that falls squarely into dev preferences, and the ability to return early - which also doesn't apply in this situation.